webapp/app/statisticalFactsheets/CarbonGrowingStock/CarbonGrowingStock.js

```diff
@@ -0,0 +1,12 @@
+import './carbonGrowingStock.less'
+import React from 'react'
+
+const CarbonGrowingStock = () => {
+  return (
+    <div className="statistical-factsheets-carbon-growing-stock">
+      <h3 className="header">Carbon and Growing Stock (1900 - 2000)</h3>
```

Comment:
Are these supposed to be `1900 -` or `1990 -`?

Reply:
that's just a template. they will need to use i18n when implemented